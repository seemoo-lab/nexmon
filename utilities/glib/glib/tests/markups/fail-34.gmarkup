<foo>&#234234</foo>