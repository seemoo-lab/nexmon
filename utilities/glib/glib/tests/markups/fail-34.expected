ELEMENT 'foo'
ERROR Error on line 1: Character reference did not end with a semicolon; most likely you used an ampersand character without intending to start an entity - escape ampersand as &amp;
